src/main/java/com/resource/management/data/Detachment.java

```diff
@@ -1,24 +1,21 @@
 package com.resource.management.data;
 
-import org.apache.commons.lang.builder.ToStringBuilder;
+import java.util.List;
+
 import org.springframework.data.annotation.Id;
 
-public class Detachment {
-    @Id
-    private final String name;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.ToString;
 
-    public Detachment(String name) {
-        this.name = name;
-    }
+@Data
+@AllArgsConstructor
+@ToString
```

Comment:
I'm new with Lombok, but do we really need ToString annotation if we have Data? I think Data already generates the toString() method.

Reply:
Yes, @Data includes @ToString.